Mathlib/GroupTheory/Subgroup/Saturated.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johan Commelin
 -/
 import Mathlib.GroupTheory.Subgroup.Basic
+import Mathlib.Algebra.Module.Basic
```

Comment:
Do you know why we need the definition of a module here? It seems this fits within group theory.